src/app/backend/resource/deployment/deploy.go

```diff
@@ -106,6 +106,18 @@ type AppDeploymentFromFileSpec struct {
 	Validate bool `json:"validate"`
 }
 
+// AppDeployRollBackSpec is a specification for deployment rollback from an specific revision number
+type AppDeployRollBackSpec struct {
+	// Name is the name of the deployment which will be rollback
+	Name string `json:"name"`
+
+	// Namespace is the namespace of the deployment which will be rollback
+	Namespace string `json:"namespace"`
```

Comment:
I'd prefer to keep name and namespace as a path params like we do it in other deployment endpoints.

Reply:
I also thought about this problem before. Because rollback is a `POST` request (because it has the operation to modify the deployment), which is different from the read data operation (`GET` request) of other deployments, so I define a structure just like in handler `/ appdeployment`.

```
	apiV1Ws.Route(
		apiV1Ws.POST("/appdeployment").
			To(apiHandler.handleDeploy).
			Reads(deployment.AppDeploymentSpec{}).
			Writes(deployment.AppDeploymentSpec{}))
```

Reply:
Post should actually store data in most of the cases. Perhaps we should use different method for that.

Reply:
Also the url should be `/deployment/{namespace}/{deployment}/rollback` and only version/revision should be passed to as post param.

Reply:
OK, I'll modify it later，thank you both~